configureWorkspace/configure.ts

```diff
@@ -637,28 +675,35 @@ export async function configure(folderPath?: string): Promise<void> {
         return createWorkspaceFileIfNotExists(fileName, DOCKER_FILE_TYPES[fileName]);
     }));
 
-    if (reporter) {
-        /* __GDPR__
-        "command" : {
-            "command" : { "classification": "SystemMetaData", "purpose": "FeatureInsight" },
-            "platformType": { "classification": "SystemMetaData", "purpose": "FeatureInsight" }
-        }
-        */
-        reporter.sendTelemetryEvent('command', {
-            command: 'vscode-docker.configure',
-            platformType
-        });
-    }
+    // Don't wait
+    vscode.window.showInformationMessage(
+        filesWritten.length ?
+            `The following files were written into the workspace:${EOL}${EOL}${filesWritten.join(', ')}` :
+            "No files were written"
+    );
 
     async function createWorkspaceFileIfNotExists(fileName: string, generatorFunction: GeneratorFunction): Promise<void> {
```

Comment:
Again - not your code - but I'm really not a fan of these nested functions. It makes it difficult to read. Specifically, it makes it seem like filesWritten is always empty when you're showing the info message above (until I realized it was being modified in this function)